build/test-list/src/main/resources/test-jars.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<testJars>
+    <testJar>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-integration-test-aws</artifactId>
+        <version>0.4.1-SNAPSHOT</version>
```

Comment:
Wouldn't be better to avoid having those version here ?

Reply:
Good point. Let me figure out a better way how to get the version on the platform side.

Reply:
can't we just use maven builtins to substitute place-holders ?

Reply:
This XML file will be consumed via a versioned jar on the platform side - so the version can be taken from the version of the jar. The rpkgtest-maven-plugin mojo for generating the test modules just needs to get adapted for that.

Reply:
Removing the commit from this PR for now. Will submit it separately.